sql-playground.sql

```diff
@@ -72,3 +50,50 @@ SELECT * FROM comments;
 -- JOIN   properties p ON p.name LIKE it.item
 -- JOIN   tags t ON t.name ILIKE ('%' || it.tag ||'%');
 
+\echo '\n articles filtered by topic mitch: \n'
+SELECT a.article_id, a.title, a.topic, a.author, a.created_at, a.votes, a.article_img_url, count(c.article_id) as comment_count
+        FROM articles a
+        JOIN comments c
+            ON a.article_id = c.article_id
+        WHERE a.topic = 'mitch'
+        GROUP BY a.article_id
+        ORDER BY a.created_at DESC;
+
+\echo '\n articles filtered by no topic: \n'
+SELECT a.article_id, a.title, a.topic, a.author, a.created_at, a.votes, a.article_img_url,count(c.article_id) AS comment_count
+        FROM articles a
+        JOIN comments c
+            ON a.article_id = c.article_id
+        GROUP BY a.article_id
+        ORDER BY a.created_at DESC;
+
+--how to do cast???
+-- \echo '\n articles filtered by no topic: \n'
+-- SELECT a.article_id, a.title, a.topic, a.author, a.created_at, a.votes, a.article_img_url, CAST(count(c.article_id) AS comment_count AS integer)
+--         FROM articles a
+--         JOIN comments c
+--             ON a.article_id = c.article_id
+--         GROUP BY a.article_id
+--         ORDER BY a.created_at DESC;
```

Comment:
Check out my comment above for a tip on how to type cast!

Reply:
I already did this on a later PR :)